src/test/java/io/debezium/connector/yugabytedb/YugabyteDBConnectorIT.java

```diff
@@ -0,0 +1,170 @@
+package io.debezium.connector.yugabytedb;
+
+import static org.fest.assertions.Assertions.*;
+
+import java.sql.SQLException;
+import java.util.concurrent.CompletableFuture;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.log4j.Logger;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.testcontainers.containers.YugabyteYSQLContainer;
+
+import io.debezium.DebeziumException;
+import io.debezium.config.Configuration;
+import io.debezium.config.Field;
+import io.debezium.embedded.AbstractConnectorTest;
+
+public class YugabyteDBConnectorIT extends AbstractConnectorTest {
```

Comment:
The class and file name has IT. Maven will consider it as an integration test. Is this different from DBCompleteTests? AFAICT, both start containers.